### PR TITLE
Adding pestpp

### PR DESCRIPTION
**Pull request by scharlton2**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by scharlton2** [comment]
@conda-forge/help-c-cpp, ready for review!

**Comment by scharlton2** [comment]
@conda-forge-admin, please lint

**Comment by scharlton2** [comment]
@conda-forge/help-c-cpp, ready for review!

**Comment by scharlton2** [comment]
### For reviewers
This is an updated submission from https://github.com/conda-forge/staged-recipes/pull/20114

I was unable to determine what the problem was with:
    * License is not an SPDX identifier (or a custom LicenseRef) nor an SPDX license expression.

I believe all SPDX identifiers are valid based on https://spdx.org/licenses/.

Thanks

**Comment by scharlton2** [comment]
@conda-forge/help-c-cpp @conda-forge/staged-recipes the PR is ready for review, thanks!

File: recipes/pestpp/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  build:
+    - {{ compiler('cxx') }}
+    - cmake >=3.9
+    - make
+    - ninja
+  host:
+    - python
+    - pip
+  run:
+    - python
```
Comment (mfansler) [inline recipes/pestpp/meta.yaml:26]: Why is Python a requirement? I see that it's used upstream in the CI for benchmarking, but does the library actually need it?

File: recipes/pestpp/build.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+mkdir build
+cd build
```
Comment (mfansler) [inline recipes/pestpp/build.sh:4]: `pushd` is preferable, and `popd` when done.

**Review by mfansler — CHANGES_REQUESTED** [review]

**Comment by scharlton2** [comment]
@jdhughes-usgs do you agree to be a recipe-maintainer?

**Comment by jdhughes-usgs** [comment]
@scharlton2 I agree to be a maintainer

File: recipes/pestpp/meta.yaml (hunk 1 above)
Comment (scharlton2) [reply recipes/pestpp/meta.yaml]: I think it was left over from a template.  I removed both of them.

File: recipes/pestpp/build.sh (hunk 2 above)
Comment (scharlton2) [reply recipes/pestpp/build.sh]: I updated the scripts

**Review by mfansler — APPROVED** [review]

**Comment by mfansler** [comment]
Thanks for submitting this!

**Comment by scharlton2** [comment]
Thanks!

**Pull request merged**